root/layout/components/sidebar/EventSidebar.js

```diff
@@ -35,13 +38,45 @@ component EventSidebar(event: EventT) {
   const $c = React.useContext(CatalystContext);
   const hasBegin = !isDateEmpty(event.begin_date);
   const hasEnd = !isDateEmpty(event.end_date);
+  const eventArtwork = $c.stash.event_artwork;
+  const eventArtPresence = event.event_art_presence;
 
   return (
     <div id="sidebar">
-      <CommonsImage
-        cachedImage={$c.stash.commons_image}
-        entity={event}
-      />
+      {(eventArtPresence === 'present' || !$c.stash.commons_image) ? (
+        <div className="event-art">
+          {eventArtPresence === 'present' && eventArtwork ? (
+            <Artwork
+              artwork={eventArtwork}
+              message={ReactDOMServer.renderToStaticMarkup(exp.l(
+                'Image failed to load correctly.' +
+                '<br/>{all|View all images}.',
+                {all: entityHref(event, 'event-art')},
+              ))}
+            />
+          ) : eventArtPresence === 'darkened' ? (
+            l(`Images for this item have been hidden
+               by the Internet Archive because of a takedown request.`)
+          ) : (
+            <p className="event-art-note" style={{textAlign: 'left'}}>
+              {eventArtPresence === 'present' ? (
+                <>
+                  {l('No poster art available.')}
```

Comment:
That sounds fine. I mainly added "art" so that "poster art" is consistent with "cover art."

Right now the CAA one is "No front cover image available"; do you think it's worth changing?

Reply:
Sorry, I did misread `ReleaseSidebar`’s code and thought that “No cover art available” was for the same case but it isn’t. No change needed for the CAA side of the force.